app/src/main/res/values-ca/strings.xml

```diff
@@ -938,10 +938,11 @@
     <string name="guest_room_link_disabled">Generating guest links is now disabled for all group admins</string>
     <!-- New Device dialog -->
     <string name="new_device_dialog_current_user_title">Your account was used on</string>
-    <string name="new_device_dialog_current_user_message">%1$s\nFrom: %2$s \n\nIf you didn\'t do this, remove the device and reset your password.</string>
+    <string name="new_device_dialog_message_defice_info">%1$s\nFrom: %2$s \n</string>
+    <string name="new_device_dialog_current_user_message">%1$s\nIf you didn\'t do this, remove the device and reset your password.</string>
     <string name="new_device_dialog_current_user_btn">Manage Devices</string>
     <string name="new_device_dialog_other_user_title">Your account “%1$s (@%2$s)” was used on:</string>
-    <string name="new_device_dialog_other_user_message">%1$s\nFrom: %2$s \n\nIf you didn\'t do this, remove the device (navigate to “Manage Devices” in the Settings section of this account), and reset your password.</string>
+    <string name="new_device_dialog_other_user_message">%1$s\nIf you didn\'t do this, remove the device (navigate to “Manage Devices” in the Settings section of this account), and reset your password.</string>
```

Comment:
Side-question, maybe aimed at design @WolfiWire.

But instead of:

> If you didn't do this, remove the device (navigate to “Manage Devices” in the Settings section of this account), and reset your password

Could we be a bit more helpful and maybe give a button to the user to perform such actions?